include/openassetio/managerAPI/ManagerInterface.hpp

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+#pragma once
+
+#include <openassetio/abi.hpp>
+
+namespace openassetio
+{
+inline namespace OPENASSETIO_VERSION
+{
+class ManagerInterface
+{
+};
+}  // namespace OPENASSETIO_VERSION
+}  // namespace openassetio
```

Comment:
No blank line at EOF?

Also, I assume no indentation is just GitHub not showing properly?

Reply:
Do you mean indentation of namespaces? Most styles I've seen in the past don't indent on namespaces, otherwise you rapidly reach your line length limit.

Reply:
Was just a question. I usually see the contents of namespaces indented.